course_info/views.py

```diff
@@ -37,6 +39,27 @@ def partials(request, path):
     return render(request, 'course_info/partials/' + path, {})
 
 
+def _get_canvas_roles():
+    """
+    Get the list of canvas roles from the canvas root account ('self') and
+    create a list in the format the front end wants.
+    :return:
+    """
+    roles = []
+    try:
+        canvas_roles = canvas_api_helpers_roles.get_roles_for_account_id('self')
+        for role_id in canvas_roles:
```

Comment:
Since it's a dict, I think we can do something like `for role_id, role in canvas_roles.items()`, and then line 54 can read
```
'roleName': role['label']
```